network/vpc/reference-content/dns.mdx

```diff
@@ -0,0 +1,79 @@
+---
+meta:
+  title: Understanding Scaleway DNS
```

Comment:
```suggestion
  title: Understanding Scaleway DNS for VPC and Private Networks
```

Maybe add this to the title to avoid confusion avec the DNS & Domains product?